README.md

```diff
@@ -1,3 +1,38 @@
-# Moeller Lab metagenomics processing pipeline
+# Moeller Lab Metagenomics Processing Pipeline
 
+## Overview
 Snakemake pipeline for basic processing of metagenomic data from the lab.
+
+Module 1: **sn-mg-QC**
+
+Inputs:
+  - Directory of raw fastq files from samples
+  - `File_Paths.txt` document, with 3 columns: `Sample_ID`, `R1`, `R2`
+    - Files will be renamed with `Sample_ID`
+    - `R1` and `R2` should be the full path to the forward and reverse reads, respectively
+  - Host genome in fasta format or bowtie2 index thereof
+  - `config.yaml` file that defines all of the parameters of the pipeline
+
+Outputs:
+  - Directory of quality filtered fastq files that have been depleted of host reads
+  - Directory of quality filtered fastq files containing only host reads
```

Comment:
Maybe BAM alignment instead? Or both?

Reply:
Ah yes! Definitely output a .BAM file for the host. I'll update this next.